src/state/account/accountService.ts

```diff
@@ -65,44 +82,30 @@ export const fetchProfile = (params, currentUser) => {
     ${userFields}
     ${userPreferences}
   `;
-  return terrasoApi
-    .requestGraphQL(query, { email: currentUser.email })
-    .then(_.get('users.edges[0].node'))
-    .then(user => user || Promise.reject('not_found'))
-    .then(user => ({
-      ..._.omit('preferences', user),
-      preferences: parsePreferences(user),
-    }));
+  const result = await terrasoApi.requestGraphQL<UserQuery>(query, {
+    email: currentUser?.email,
+  });
+
+  const user = result.users.edges.at(0);
+  if (user === undefined) {
```

Comment:
they are falsy, i just have a style preference for explicitly checking for equality to `undefined` so it's clear what is actually being checked for. Typescript will raise an IDE/compile error if you try to do anything with a `null` or `undefined` type, and will also raise a warning if you do an unnecessary check. so in this case it can actually be concluded from the fact that this all type checks that the `at` function can return undefined values but not null values (`at` doesn't raise an exception if you access something that doesn't exist, just returns undefined).

the logic is changing because Typescript is not very good at typing big lodash expressions. it can often do it, and i tried to keep with lodash where possible to keep the PR smaller, but in this case it was cleaner to just break it out. i do think this code is easier to understand for someone new to the codebase, but i know there's always a tradeoff with lodash's succinctness/robustness. i do think that with Typescript there's less of a tradeoff since the types can provide the robustness, i.e. in a sense both Typescript and lodash are smart enough to handle the null and undefined values, just one does it implicitly and one explicitly